### DISCUSSION TITLE
Grammarly extension causes snapshot missing issue on <textarea> inputs

### DISCUSSION THREAD

**Discussion by JamesDavis7** [livewire/livewire#8519] [7. Bugs (archived)]
I spent the past two nights working on my TALL stack task tracking app, which includes a textarea for creating and updating tasks. I encountered an issue where, after updating a task description and hitting "update," I received the error: "Uncaught Snapshot missing on Livewire component with id:".

The problem was due to Grammarly adding a 'spellcheck' attribute to the textarea whenever I typed in it, indicated by the Grammarly icon appearing in the bottom right of the textarea box. This caused the textarea input to differ from its initial snapshot.

I believe this is a bug because Livewire should not be storing this sort of data in the snapshot information during hydration.

**Comment by jbajou** [comment]
We're experiencing exactly the same issue and this is highly uncomfortable to debug as we could not reproduce until we understood that the customer was using grammarly.
@JamesDavis7 did you find a solution ?

**Reply by JamesDavis7** [reply]
The app I was building was more for personal practice and never made it to any end users. 

To continue developing, I simply uninstalled Grammarly from my own personal extensions then it worked.

Sorry to say I'm unaware of a "production-grade" solution as of now and I'm very surprised maintainers haven't addressed this.

**Comment by chatisk** [comment]
There is a way to disable grammarly on these fields by adding the `data-enable-grammarly="false"` tag on your html elements

But I know it's a workaround

**Reply by JamesDavis7** [reply]
Nice, for now it's better than nothing I suppose.

**Reply by chatisk** [reply]
Yeah unfortunately. It's polling in the background. And somehow Livewire thinks its own HTTP request. (I guess). And that confuses LW totally
